build.gradle

```diff
@@ -985,6 +986,13 @@ task javadocJar(type: Jar, dependsOn: ["javadoc"]) {
 	archiveClassifier = 'javadoc'
 }
 
+generatePackageInfoMappings {
+	dependsOn mapIntermediaryJar
+	inputJar = intermediaryJar
+	packageName = "net/minecraft/util/packageinfo/"
+	outputDir = file("mappings/net/minecraft/util/packageinfo")
```

Comment:
Not really sure if `util` is the best place for these. That's a package for actual utility classes, not a dump for unused things.

Reply:
Yeah... this was up in the air, do you have any suggestions?

Reply:
Maybe even `net.minecraft.unused.packageinfo`? These classes shouldn't really be seeing any use (unless Yarn switches to follow Mojang's package structure :yeefuckinhaw:).

Reply:
That makes more sense yes, its clear they wont be used, I will move them 👍

Reply:
imo util can just be unused.